include/vcpkg/base/messages.h

```diff
@@ -314,7 +314,7 @@ namespace vcpkg::msg
 
     inline void print_warning(const LocalizedString& s)
     {
-        print(Color::warning, format(msgErrorMessage).append(s).appendnl());
```

Comment:
OMFG

Reply:
copy-paste errors like this happen to all of us, and nobody caught it in review either